raddb/mods-available/ldap

```diff
@@ -135,231 +191,287 @@ ldap {
 #		&reply:Tunnel-Medium-Type	:= 'radiusTunnelMediumType'
 #		&reply:Tunnel-Private-Group-ID	:= 'radiusTunnelPrivategroupId'
 
-		#  Where only a list is specified as the RADIUS attribute,
+		#  NOTE: Where only a list is specified as the RADIUS attribute,
 		#  the value of the LDAP attribute is parsed as a valuepair
 		#  in the same format as the 'valuepair_attribute' (above).
 		&control:			+= 'radiusControlAttribute'
 		&request:			+= 'radiusRequestAttribute'
 		&reply:				+= 'radiusReplyAttribute'
 	}
 
-	#  Set to yes if you have eDirectory and want to use the universal
+	#
+	#  edir:: Set to `yes` if you have eDirectory and want to use the universal
 	#  password mechanism.
+	#
 #	edir = no
 
-	#  Set to yes if you want to bind as the user after retrieving the
-	#  Cleartext-Password. This will consume the login grace, and
-	#  verify user authorization.
-#	edir_autz = no
-
-	#  Note: set_auth_type was removed in v3.x.x
-	#  Equivalent functionality can be achieved by adding the following
-	#  stanza to the recv Access-Request {} section of your virtual server.
 	#
-	#    ldap
-	#    if ((ok || updated) && User-Password) {
-	#        update {
-	#            control:Auth-Type := ldap
-	#        }
-	#    }
+	#  edir_autz:: Set to `yes` if you want to bind as the user after retrieving the
+	#  `Cleartext-Password`. This will consume the login grace, and verify user authorization.
+	#
+#	edir_autz = no
 
 	#
-	#  User object identification.
+	#  ### User object identification
 	#
 	user {
-		#  Where to start searching in the tree for users
+		#
+		#  base_dn:: Where to start searching in the tree for users.
+		#
 		base_dn = "${..base_dn}"
 
-		#  Filter for user objects, should be specific enough
+		#
+		#  filter:: Filter for user objects, should be specific enough
 		#  to identify a single user object.
+		#
 		filter = "(uid=%{%{Stripped-User-Name}:-%{User-Name}})"
 
-		#  SASL parameters to use for user binds
+		#
+		#  sasl { ... }:: SASL parameters to use for user binds
 		#
 		#  When we're prompted by the SASL library, these control
 		#  the responses given.
 		#
-		#  Any of the config items below may be an attribute ref
+		#  NOTE: Any of the config items below may be an attribute ref
 		#  or and expansion.  This allows different SASL mechs,
 		#  proxy IDs and realms to be used for different users.
 		#
 		sasl {
-			# SASL mechanism
+			#
+			#  mech:: SASL mechanism.
+			#
 #			mech = 'PLAIN'
 
-			# SASL authorisation identity to proxy.
+			#
+			#  proxy:: SASL authorisation identity to proxy.
+			#
 #			proxy = &User-Name
 
-			# SASL realm. Used for kerberos.
+			#
+			#  realm:: SASL realm. Used for kerberos.
+			#
 #			realm = 'example.org'
 		}
 
-		#  Search scope, may be 'base', 'one', sub' or 'children'
+		#
+		#  scope:: Search scope, may be `base`, `one`, `sub' or `children`.
+		#
 #		scope = 'sub'
 
-		#  Server side result sorting
+		#
+		#  sort_by:: Server side result sorting.
+		#
+		#  [NOTE]
```

Comment:
I don't think this needs to be a note.  It's just describing the format of the attribute string.